python/desc/imsim/imSim.py

```diff
@@ -371,6 +395,11 @@ def sources_from_list(object_lines, obs_md, phot_params, file_name):
 
     return gs_object_arr, out_obj_dict
 
+def get_image_dirs():
+    """
+    Return a list of possible directories for FITS images.
+    """
+    return os.environ.get('IMAGE_DIR_PATH', '.').split(':')
```

Comment:
This gives you image_dir_path or '.' if it is not set.  Is this consistent with the command string?

"This will be prepended to any existing IMAGE_DIR_PATH environment variable, for which $CWD is included by default." ?

I think the code may leave out CWD if IMAGE_DIR_PATH is set.

Reply:
Yes, that's true.  I realized that when I first wrote it, but didn't get around to implementing it properly.  I'll fix it.